content/en/docs/setup/cri/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Kubernetes CRI
+weight: 30
+---
```

Comment:
Are more topics planned for CRI? If not, we should have a single "CRI Installation" topic instead of nesting only one topic.